<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTableWidget" name="timeTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>192</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ActionButton" name="leftArrow">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ActionButton" name="rightArrow">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="info">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="newSchedule"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionSchedule"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionRecolour"/>
   </widget>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="actionBy_Empty_Hours"/>
    <addaction name="actionBy_Days_Off"/>
    <addaction name="actionReverse_Days_Off"/>
    <addaction name="actionReverse_Empty_Hours"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuSort"/>
  </widget>
  <action name="newSchedule">
   <property name="text">
    <string>New Schedule</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load in data to the table&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSchedule">
   <property name="text">
    <string>Schedule</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionRecolour">
   <property name="text">
    <string>Recolour</string>
   </property>
  </action>
  <action name="actionBy_Empty_Hours">
   <property name="text">
    <string>By Empty Hours</string>
   </property>
  </action>
  <action name="actionBy_Days_Off">
   <property name="text">
    <string>By Days Off</string>
   </property>
  </action>
  <action name="actionReverse_Days_Off">
   <property name="text">
    <string>Reverse Days Off</string>
   </property>
  </action>
  <action name="actionReverse_Empty_Hours">
   <property name="text">
    <string>Reverse Empty Hours</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ActionButton</class>
   <extends>QPushButton</extends>
   <header>actionbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
